OKD.md

```diff
@@ -0,0 +1,85 @@
+# [kubevirtci](README.md): Getting Started with multi-node OKD Provider
+
+Download this repo
+```
+git clone https://github.com/kubevirt/kubevirtci.git
+cd kubevirtci
+```
+
+Start okd cluster (pre-configured with a master and worker node)
+```
+export KUBEVIRT_PROVIDER=okd-4.1
```

Comment:
okd 4.1 is broken, current we use mainly ocp-4.3 , so maybe better to refer it ?

Reply:
on the other hand, since ocp need quay access, maybe its good to be on okd